chapters/annotations.tex

```diff
@@ -2318,14 +2318,15 @@ \subsection{Licensing}\label{licensing}
 record Protection
   /*literal*/ constant Access access;
   /*literal*/ constant String features[:] "Required license features";
-
-  record License
-    /*literal*/ constant String libraryKey;
-    /*literal*/ constant String licenseFile = "" "Optional, default mapping if empty";
-  end License;
+  /*literal*/ constant IntLicense License;
```

Comment:
I find the `Int` prefix confusing.  How about `ProtectionLicense` or `UserLicense`?